### ISSUE TITLE
Deprecated warning in pander.sessionInfo

### ISSUE THREAD

**Issue by BrianDiggs** [Rapporter/pander#172]
The following code

```r
library("pander")
pander(sessionInfo())
```

gives the warnings

```r
Warning messages:
1: 'pander.return' is deprecated.
Use 'pander_return' instead.
See help("Deprecated") 
2: 'pander.return' is deprecated.
Use 'pander_return' instead.
See help("Deprecated") 
3: 'pander.return' is deprecated.
Use 'pander_return' instead.
See help("Deprecated") 
```

This only happens with the default `compact = TRUE`, `pander(sessionInfo(), compact = FALSE)` throws no warnings.

```r
R version 3.2.0 (2015-04-16)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] pander_0.5.2

loaded via a namespace (and not attached):
[1] tools_3.2.0  Rcpp_0.11.6  digest_0.6.8
```

**Comment by daroczig** [comment]
Thank you very much for this report. Pinging @RomanTsegelskyi as well.
This will be resolved by #171, hopefully in a week or so on GitHub.

**Comment by daroczig** [comment]
Was fixed via #178 in the dev version.
